### PR TITLE
feat: support .redocly.yaml for options for redoc-cli

### LINKED ISSUE #1785: Support .redocly.yaml for options for redoc-cli
Hello,

When I preview docs with `openapi preview-docs`, it uses the theme defined in my `.redocly.yaml` file. Is there a way to render a static HTML file, much like `redoc-cli bundle`, that reads the theme from that file?

### PR DESCRIPTION
**Pull request by anastasiia-developer**
## What/Why/How?

Support .redocly.yaml for options for redoc-cli

## Reference
close #1785 

## Check yourself

- [ ] Code is linted
- [ ] Tested
- [ ] All new/updated code is covered with tests

### CODE REVIEW

**Comment by zalesky** [comment]
Would be great if there will be one more test that shows `disableSearch: true` is applied

**Comment by anastasiia-developer** [comment]
@zalesky 
Done

**Review by zalesky — APPROVED** [review]

File: cli/index.ts [hunk 1]
```diff
@@ -447,6 +453,17 @@ function getObjectOrJSON(options) {
         handleError(e);
       }
     default:
+      const configFile = findConfig();
+      if (configFile) {
+        console.log(`Found ${configFile} and use options from features.openapi`);
```
Comment (adamaltman) [inline cli/index.ts:458]:
```suggestion
        console.log(`Found ${configFile} and using features.openapi options`);
```

**Comment by adamaltman** [comment]
Is there any docs? How would someone use this?

File: cli/README.md [hunk 2]
```diff
@@ -31,4 +31,10 @@ Some examples:
 - Bundle using a custom template and add custom `templateOptions`:<br/>
   `$ redoc-cli build [spec] -t custom.hbs --templateOptions.metaDescription "Page meta description"`
 
+#### With `.redocly.yaml` file ([more info](https://redoc.ly/docs/api-reference-docs/on-premise/cli-usage/#example-with-try-it-console-enabled)):
```
Comment (anastasiia-developer) [inline cli/README.md:34]:
```suggestion
#### With `.redocly.yaml` file ([more info](https://redocly.com/docs/cli/configuration/#redocly-configuration-file)):
```

**Review by AlexVarchuk — APPROVED** [review]

File: cli/README.md [hunk 3]
```diff
@@ -31,4 +31,10 @@ Some examples:
 - Bundle using a custom template and add custom `templateOptions`:<br/>
   `$ redoc-cli build [spec] -t custom.hbs --templateOptions.metaDescription "Page meta description"`
 
+#### With `.redocly.yaml` file ([more info](https://redocly.com/docs/cli/configuration/#redocly-configuration-file)):
+
+1. Go to folder with `.redocly.yaml` and your definition file
```
Comment (adamaltman) [inline cli/README.md:36]: Does it work with `redocly.yaml` file too?

Comment (anastasiia-developer) [reply cli/README.md:36]: Yes, it works too

Comment (adamaltman) [reply cli/README.md:36]:
```suggestion
1. Go to folder with your Redocly configuration file (`.redocly.yaml` or `redocly.yaml`) and your OpenAPI definition file.
```

File: cli/README.md [hunk 4]
```diff
@@ -31,4 +31,10 @@ Some examples:
...
+2. Build site using command (options from `.redocly.yaml` file will be automatically fetched):
```
Comment (adamaltman) [inline cli/README.md:37]:
```suggestion
2. Build the site using the `build` command (options from the Redocly configuration file will be automatically fetched):  
```

File: cli/README.md [hunk 5]
```diff
@@ -31,4 +31,10 @@ Some examples:
 - Bundle using a custom template and add custom `templateOptions`:<br/>
   `$ redoc-cli build [spec] -t custom.hbs --templateOptions.metaDescription "Page meta description"`
 
+#### With `.redocly.yaml` file ([more info](https://redocly.com/docs/cli/configuration/#redocly-configuration-file)):
```
Comment (adamaltman) [inline cli/README.md:34]:
```suggestion
#### With Redocly configuration file ([more info](https://redocly.com/docs/cli/configuration/#redocly-configuration-file)):
```
(I missed a spot)

**Review by adamaltman — APPROVED** [review]
(From the docs perspective.)

**Pull request merged**